src/components/ParalaxBlock/index.css

```diff
@@ -1,88 +1,52 @@
 .card1,
```

Comment:
this PR is pretty similar to 
https://github.com/ReactTeam13/culture-portal/pull/58/commits/26e4f5c1a50df181a0ac9d6e3e459bcfa58bcd1c
Are you sure you need both?